homeassistant/brands/esera.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We should not use brands in case there is just one integration. Please remove the brand.

Reply:
They also have eBus devices, so I thought there should be a brand in case eBus gets tested/added. See https://esera.de/produkte/ebus/. Fine for me to remove the brand, though.

Reply:
> They also have eBus devices

That is cool, but those are not in Home Assistant.